### PR TITLE
feat(atenlib):add ops(scalar tensor, constant_pad_nd)

### PR DESCRIPTION
**Pull request by xiaowuhu**

### CODE REVIEW

File: onnxscript/function_libs/torch_aten/ops/core.py
Comment (gramalingam) [inline onnxscript/function_libs/torch_aten/ops/core.py:1094]: Why does this need to be trace-only?

File: onnxscript/function_libs/torch_aten/ops/core.py
Comment (gramalingam) [inline onnxscript/function_libs/torch_aten/ops/core.py:1095]: What is the type of pad supposed to be? `Sequence[INT64]` looks unlikely. Is it really expected to be a list of tensors (containing a single element)? Or, it is expected to be a tensor? Or, a list of ints?

`INT64` indicates a tensor (of rank 0) of int64 value.

Comment (xiaowuhu) [reply onnxscript/function_libs/torch_aten/ops/core.py:1096]: should be INT64

File: onnxscript/function_libs/torch_aten/ops/core.py
Comment (xiaowuhu) [reply onnxscript/function_libs/torch_aten/ops/core.py]: Thanks for the reminder. I made it with full onnx ops.

File: onnxscript/function_libs/torch_aten/ops/core.py
Comment (justinchuby) [inline onnxscript/function_libs/torch_aten/ops/core.py:1111]:
```suggestion
    zero_count = op.Sub(op.Mul(rank, 2), op.Size(pad))
```

for simplicity

File: onnxscript/function_libs/torch_aten/ops/core.py
Comment (justinchuby) [inline onnxscript/function_libs/torch_aten/ops/core.py:1112]:
```suggestion
```
is reshape needed?

File: onnxscript/function_libs/torch_aten/ops/core.py
Comment (justinchuby) [inline onnxscript/function_libs/torch_aten/ops/core.py:1130]:
```suggestion
    return op.Pad(self, onnx_padding, value)
```

for simplicity

**Review by justinchuby — COMMENTED** [review]
Looking great, thank you! Some minor comments

File: onnxscript/function_libs/torch_aten/ops/core.py
Comment (justinchuby) [reply onnxscript/function_libs/torch_aten/ops/core.py:1112]: my bad

**Review by justinchuby — APPROVED** [review]

**Comment by justinchuby** [comment]
Will be great to get a second review from @fatcat-z

**Review by fatcat-z — APPROVED** [review]
Thank you!

**Pull request merged**
